[2025-01-07T08:56:16.132+0000] {processor.py:161} INFO - Started process (PID=29503) to work on /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:56:16.143+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2025-01-07T08:56:16.149+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:56:16.148+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:56:16.187+0000] {processor.py:840} INFO - DAG(s) 'example_branch_datetime_operator_3', 'example_branch_datetime_operator_2', 'example_branch_datetime_operator' retrieved from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:56:16.304+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:56:16.303+0000] {dag.py:3047} INFO - Sync 3 DAGs
[2025-01-07T08:56:16.380+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:56:16.379+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:56:16.399+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:56:16.399+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:56:16.408+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:56:16.407+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_3 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:56:16.480+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 0.355 seconds
[2025-01-07T08:57:44.975+0000] {processor.py:161} INFO - Started process (PID=29578) to work on /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:57:44.978+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2025-01-07T08:57:44.979+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:57:44.979+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:57:44.998+0000] {processor.py:840} INFO - DAG(s) 'example_branch_datetime_operator_3', 'example_branch_datetime_operator_2', 'example_branch_datetime_operator' retrieved from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:57:45.050+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:57:45.050+0000] {dag.py:3047} INFO - Sync 3 DAGs
[2025-01-07T08:57:45.117+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:57:45.116+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:57:45.133+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:57:45.132+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:57:45.138+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:57:45.138+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_3 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:57:45.190+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 0.220 seconds
[2025-01-07T08:58:15.902+0000] {processor.py:161} INFO - Started process (PID=29645) to work on /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:58:15.905+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2025-01-07T08:58:15.909+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:58:15.907+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:58:15.931+0000] {processor.py:840} INFO - DAG(s) 'example_branch_datetime_operator_3', 'example_branch_datetime_operator_2', 'example_branch_datetime_operator' retrieved from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:58:15.992+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:58:15.992+0000] {dag.py:3047} INFO - Sync 3 DAGs
[2025-01-07T08:58:16.050+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:58:16.049+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:58:16.058+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:58:16.057+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:58:16.062+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:58:16.062+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_3 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:58:16.089+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 0.194 seconds
[2025-01-07T08:58:53.699+0000] {processor.py:161} INFO - Started process (PID=29714) to work on /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:58:53.702+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2025-01-07T08:58:53.704+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:58:53.703+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:58:53.755+0000] {processor.py:840} INFO - DAG(s) 'example_branch_datetime_operator', 'example_branch_datetime_operator_2', 'example_branch_datetime_operator_3' retrieved from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T08:58:53.898+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:58:53.897+0000] {dag.py:3047} INFO - Sync 3 DAGs
[2025-01-07T08:58:53.993+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:58:53.993+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:58:54.022+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:58:54.022+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:58:54.029+0000] {logging_mixin.py:188} INFO - [2025-01-07T08:58:54.029+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_3 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T08:58:54.091+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 0.407 seconds
[2025-01-07T09:02:22.595+0000] {processor.py:161} INFO - Started process (PID=29802) to work on /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T09:02:22.623+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2025-01-07T09:02:22.631+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:02:22.630+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T09:02:22.697+0000] {processor.py:840} INFO - DAG(s) 'example_branch_datetime_operator_3', 'example_branch_datetime_operator_2', 'example_branch_datetime_operator' retrieved from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T09:02:22.921+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:02:22.920+0000] {dag.py:3047} INFO - Sync 3 DAGs
[2025-01-07T09:02:23.055+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:02:23.051+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T09:02:23.089+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:02:23.088+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T09:02:23.109+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:02:23.099+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_3 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T09:02:23.245+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 0.697 seconds
[2025-01-07T09:06:52.625+0000] {processor.py:161} INFO - Started process (PID=29892) to work on /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T09:06:52.628+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2025-01-07T09:06:52.631+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:06:52.630+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T09:06:52.668+0000] {processor.py:840} INFO - DAG(s) 'example_branch_datetime_operator_3', 'example_branch_datetime_operator_2', 'example_branch_datetime_operator' retrieved from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T09:06:53.388+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:06:53.378+0000] {dag.py:3047} INFO - Sync 3 DAGs
[2025-01-07T09:06:53.565+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:06:53.564+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T09:06:53.602+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:06:53.601+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T09:06:53.628+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:06:53.617+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_3 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T09:06:53.813+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 1.211 seconds
[2025-01-07T09:28:04.390+0000] {processor.py:161} INFO - Started process (PID=30053) to work on /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T09:28:04.616+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2025-01-07T09:28:05.738+0000] {logging_mixin.py:188} INFO - [2025-01-07T09:28:05.403+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T09:28:23.319+0000] {processor.py:840} INFO - DAG(s) 'example_branch_datetime_operator_2', 'example_branch_datetime_operator', 'example_branch_datetime_operator_3' retrieved from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T11:36:25.066+0000] {processor.py:161} INFO - Started process (PID=254) to work on /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T11:36:25.336+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2025-01-07T11:36:25.731+0000] {logging_mixin.py:188} INFO - [2025-01-07T11:36:25.646+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T11:36:55.047+0000] {processor.py:840} INFO - DAG(s) 'example_branch_datetime_operator_3', 'example_branch_datetime_operator_2', 'example_branch_datetime_operator' retrieved from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T12:28:37.476+0000] {processor.py:161} INFO - Started process (PID=172) to work on /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T12:28:37.487+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2025-01-07T12:28:37.885+0000] {logging_mixin.py:188} INFO - [2025-01-07T12:28:37.775+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T12:28:40.108+0000] {processor.py:840} INFO - DAG(s) 'example_branch_datetime_operator_2', 'example_branch_datetime_operator', 'example_branch_datetime_operator_3' retrieved from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T12:28:59.847+0000] {logging_mixin.py:188} INFO - [2025-01-07T12:28:59.770+0000] {dag.py:3047} INFO - Sync 3 DAGs
[2025-01-07T12:29:08.244+0000] {logging_mixin.py:188} INFO - [2025-01-07T12:29:08.230+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T12:29:10.216+0000] {logging_mixin.py:188} INFO - [2025-01-07T12:29:10.167+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_2 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T12:29:10.248+0000] {logging_mixin.py:188} INFO - [2025-01-07T12:29:10.248+0000] {dag.py:3834} INFO - Setting next_dagrun for example_branch_datetime_operator_3 to 2025-01-06 00:00:00+00:00, run_after=2025-01-07 00:00:00+00:00
[2025-01-07T12:29:14.820+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py took 37.424 seconds
[2025-01-07T12:34:31.709+0000] {processor.py:161} INFO - Started process (PID=267) to work on /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T12:34:32.111+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py for tasks to queue
[2025-01-07T12:34:33.219+0000] {logging_mixin.py:188} INFO - [2025-01-07T12:34:32.788+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T12:34:47.852+0000] {processor.py:840} INFO - DAG(s) 'example_branch_datetime_operator_3', 'example_branch_datetime_operator_2', 'example_branch_datetime_operator' retrieved from /home/airflow/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_datetime_operator.py
[2025-01-07T12:35:43.996+0000] {processor.py:186} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/impl.py", line 145, in _do_get
    with util.safe_reraise():
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/impl.py", line 143, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 690, in __connect
    with util.safe_reraise():
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.10/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "postgres" to address: Temporary failure in name resolution


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/dag_processing/processor.py", line 182, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/dag_processing/processor.py", line 163, in _handle_dag_file_processing
    result: tuple[int, int] = dag_file_processor.process_file(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/dag_processing/processor.py", line 859, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/api_internal/internal_api_call.py", line 114, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/dag_processing/processor.py", line 895, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 659, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
  File "/home/airflow/.local/lib/python3.10/site-packages/tenacity/__init__.py", line 347, in __iter__
    do = self.iter(retry_state=retry_state)
  File "/home/airflow/.local/lib/python3.10/site-packages/tenacity/__init__.py", line 325, in iter
    raise retry_exc.reraise()
  File "/home/airflow/.local/lib/python3.10/site-packages/tenacity/__init__.py", line 158, in reraise
    raise self.last_attempt.result()
  File "/usr/local/lib/python3.10/concurrent/futures/_base.py", line 451, in result
    return self.__get_result()
  File "/usr/local/lib/python3.10/concurrent/futures/_base.py", line 403, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 672, in _sync_to_db
    _serialize_dag_capturing_errors(dag, session, processor_subdir)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 637, in _serialize_dag_capturing_errors
    dag_was_updated = SerializedDagModel.write_dag(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/serialized_dag.py", line 157, in write_dag
    if session.scalar(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1747, in scalar
    return self.execute(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1716, in execute
    conn = self._connection_for_bind(bind)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1555, in _connection_for_bind
    return self._transaction._connection_for_bind(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 750, in _connection_for_bind
    conn = bind.connect()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/future/engine.py", line 412, in connect
    return super(Engine, self).connect()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 3325, in connect
    return self._connection_cls(self, close_with_result=close_with_result)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
    else engine.raw_connection()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 3404, in raw_connection
    return self._wrap_pool_connect(self.pool.connect, _connection)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 3374, in _wrap_pool_connect
    Connection._handle_dbapi_exception_noconnection(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2208, in _handle_dbapi_exception_noconnection
    util.raise_(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/impl.py", line 145, in _do_get
    with util.safe_reraise():
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/impl.py", line 143, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 690, in __connect
    with util.safe_reraise():
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.10/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8)
